src/controller/c/supervisor.c

```diff
@@ -380,6 +391,9 @@ static bool allow_search_in_proto = false;
 static const char *node_def_name = NULL;
 static int proto_id = -1;
 static const char *requested_field_name = NULL;
+static bool requested_field_count = false;
```

Comment:
I would rename this so that it is more self-explanatory otherwise it may be confused with the number of items in a MF field.
```suggestion
static bool requested_node_field_count = false;
```